docker-compose.yml

```diff
@@ -52,5 +52,56 @@ services:
     volumes:
       - /app/node_modules
       - ./app:/app
+  redis:
+    image: "redis:alpine"
+    command: redis-server
+    volumes:
+     - ./mockDB/redis-data:/var/lib/redis
+  celery:
+    depends_on:
+      - redis
+      - fuseki
+    volumes:
+      - ./mockDB/fuseki:/app/run
+      - ./whyis/materialsmine:/app
+    command: /opt/venv/bin/celery -A wsgi.celery worker -l INFO -c 4 --uid=nobody --gid=nogroup
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    build: whyis
+  celerybeat:
+    depends_on:
+      - redis
+      - fuseki
+    volumes:
+      - ./mockDB/fuseki:/app/run
+      - ./whyis/materialsmine:/app
+    command: /opt/venv/bin/celery -A wsgi.celery beat -l INFO --uid=nobody --gid=nogroup
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    build: whyis
+  fuseki:
+    command:  /opt/venv/bin/fuseki-server --mem /ds #/opt/venv/bin/whyis fuseki
+    environment:
+      - CHOKIDAR_USEPOLLING=true
+    build: whyis
+#    ports:
+#      - '3030:3030'
+    volumes:
+      - ./materialsmine:/app
+      - ./whyis/materialsmine:/app
```

Comment:
Adding services to compose any image should be avoided. I noticed these services exist without any images and in each case, we are rebuilding whyis multiple times because of the `build:` in each. I'll suggest getting rid of it and also `-CHOKIDAR_USEPOLLING=true` if you don't really need it. All the mounts can be under WHYIS. There is a decoupled Fuseki service that is readily available in the develop branch. Decoupling Fuseki is one of the keys from inside whyis is specifically requested.

Reply:
Good points, although the image only gets built once. We can also switch over to a released image too (tetherless-world/whyis is already updated with a working image). I really don't know what best practices are for docker, but this worked and seemed efficient.

Reply:
Great! I'll suggest using the released image. The implementation will be similar to Redis since it's already a built image. Would be a great idea to reach out to @stouffers if you need help with setting that up.

Reply:
I'm adding the release image to the dockerfile, but we do still need to customize it with a pip install (the whyis units converter, which isn't standard).

Reply:
If that's holding this back, I can update it

Reply:
no, it's more that it needs to be installed than that it's broken.